docs/rules/no_empty_enum.md

```diff
@@ -0,0 +1,23 @@
+Disallows the declaration of an empty enum
+
+An enum with no members serves no purpose. This rule will capture
+these situations as either unnecessary code or a mistaken empty implementation.
+
+### Invalid:
+
+```typescript
+enum Foo {}
+const enum Foo {}
```

Comment:
Deno doesn't support `const enum`, I suggest to remove this from example as it might be confusing

Reply:
thx, fixed